.buildkite/commands/release-build-wordpress.sh

```diff
@@ -19,4 +19,7 @@ echo "--- :closed_lock_with_key: Installing Secrets"
 bundle exec fastlane run configure_apply
 
 echo "--- :hammer_and_wrench: Building"
-bundle exec fastlane build_and_upload_app_store_connect skip_confirm:true create_gh_release:true
+bundle exec fastlane build_and_upload_app_store_connect \
+  skip_confirm:true \
+  create_gh_release:true \
+  beta_release:${1:-true} # use first call param, default to true for safety
```

Comment:
The alternative could be to make `$1` required and check for it at the start of the script.

This seemed more concise, and I didn't feel we needed refined input validation given these scripts are only meant to run via CI.